### ISSUE TITLE
Markup has two header landmark elements

### ISSUE THREAD

**Issue by melanie-exygy** [civiform/civiform#8672]
**Describe the bug**
Our markup includes two `<header>` landmark elements.  Although this doesn't directly violate any HTML5 rules, it is discouraged because there is a rule that the equivalent ARIA `banner` role shouldn't be used more than once.  [Deque University article](https://dequeuniversity.com/rules/axe/4.1/landmark-no-duplicate-banner). We should try to change the element type of the `<header>` that is within the USWDS government banner.  If this is difficult, we should add `aria-label` to each of the elements to distinguish between them.

![image](https://github.com/user-attachments/assets/6f44ea83-d5f4-4551-95c3-589aca29d093)

**Comment by tallulahkay** [comment]
This needs to be fixed in north star and. To fix this, change the northstar implementations to use a div rather than a header for the government banner.
